SPECS/core-packages/core-packages.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Metapackage with core sets of packages
 Name:           core-packages
-Version:        2.0
-Release:        8%{?dist}
+Version:        3.0
```

Comment:
I've been wondering if we should move this to semantic versioning

Reply:
Sure, let me convert this to an `RFC`